src/collapse/perf.rs

```diff
@@ -202,213 +221,56 @@ impl Collapse for Folder {
         None
     }
 
-    #[cfg(test)]
+    fn is_end_of_stack(&self, line: &[u8]) -> bool {
+        line.iter().all(|b| (*b as char).is_whitespace())
```

Comment:
Not sure, but would
```rust
line.trim(char::is_whitespace).is_empty()
```
be nicer here?

Reply:
We're working on a `&[u8]`, not a `&str'; so there is not `trim` method.